makeflow/src/makeflow.c

```diff
@@ -126,6 +126,7 @@ static int skip_file_check = 0;
 
 static int cache_mode = 1;
 
+static int json_input = 0;
 static int jx_input = 0;
 static char *jx_context = NULL;
```

Comment:
If these variables are only used in main function, move these to the beginning of that function. If there is a foreseeable reason to have them as global statics just let me know.